ui/raidboss/data/05-shb/eureka/delubrum_reginae_savage.js

```diff
@@ -309,6 +472,61 @@ export default {
         return { infoText: output.avoidCleave() };
       },
     },
+    {
+      id: 'DelubrumSav Lord Fateful Words',
+      // 97E: Wanderer's Fate, Pushes outward on Fateful Word cast
+      // 97F: Sacrifice's Fate, Pulls to middle on Fateful Word cast
+      // Labyrinthine Fate is cast and 1 second later debuffs are applied
+      // First set of debuffs go out 7.7 seconds before Fateful Word is cast
+      // Remaining set of debuffs go out 24.3 seconds before Fateful Word is cast
+      netRegex: NetRegexes.gainsEffect({ effectId: '97[EF]' }),
+      condition: (data, matches) => {
+        return (matches.target === data.me);
+      },
+      delaySeconds: (data, matches) => {
+        if (data.secondFates)
+          return 21;
+        data.secondFates = 1;
+        return 4;
+      },
+      response: (data, matches, output) => {
+        // cactbot-builtin-response
+        output.responseOutputStrings = {
+          getOut: Outputs.out,
+          getIn: Outputs.in,
+        };
+
+        if (matches.effectId === '97F')
+          return { alertText: output.getOut() };
+        else if (matches.effectId === '97E')
+          return { alertText: output.getIn() };
+        return;
```

Comment:
```suggestion
```

Unnecessary return?

Reply:
It seemed unlikely for the branching to execute that return, but in case it did somehow I didn't know if an implied return would work. Could a player possibly have a debuff, then die and respawn before the 21 second delaySeconds go off and cause an error here without return?

Reply:
Nevermind, I guess in that case the effectId is stored from what they had before they died.